### PR TITLE
Initial work - debian-base image and README

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: debian-base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM debian:8.3
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+# Install utility scripts
+ADD ./scripts /scripts
+ENV PATH $PATH:/scripts
+
+# Download dinit (dumb-init)
+ENV DINIT_VERSION "1.0.0"
+RUN apt-get-install.sh ca-certificates curl #&& \
```
Comment (jerith) [inline debian-base/Dockerfile]: Is the `&& \` at the end of the line commented out?

Comment (JayH5) [inline debian-base/Dockerfile]: Oops

**Comment by JayH5** [comment]
@jerith: Fixed that stray `#`

**Comment by jerith** [comment]
:+1:

**Pull request merged**